integration-test/Makefile

```diff
@@ -67,4 +67,4 @@ build-with-docker: download-cbd docker-build runtest
 
 docker-build:
 	@echo "\n\033[1;96m--- build cloudbreak in docker container\033[0m\n"
-	docker run -i --rm $(ADDITIONAL_DOCKER_BUILD_COMMAND) -v $(shell pwd)/../:/tmp/prj:rw java:openjdk-8 /tmp/prj/gradlew -b /tmp/prj/build.gradle clean build
+	docker run -i --rm $(ADDITIONAL_DOCKER_BUILD_COMMAND) -v $(shell pwd)/../:/tmp/prj:rw java:openjdk-8 /tmp/prj/gradlew -b /tmp/prj/build.gradle clean build -x test
```

Comment:
In circle.yml there is a 'make build' which is always runs all unit tests, this is just a speed-up